main.tf

```
@@ -9,11 +9,12 @@ locals {
  * Create ECR repo
  */
 module "ecr" {
-  source              = "github.com/silinternational/terraform-modules//aws/ecr?ref=3.5.0"
-  repo_name           = local.app_name_and_env
-  ecsInstanceRole_arn = data.terraform_remote_state.common.outputs.ecsInstanceRole_arn
-  ecsServiceRole_arn  = data.terraform_remote_state.common.outputs.ecsServiceRole_arn
-  cd_user_arn         = data.terraform_remote_state.common.outputs.codeship_arn
+  source                = "github.com/silinternational/terraform-modules//aws/ecr?ref=8.0.1"
+  repo_name             = local.app_name_and_env
+  ecsInstanceRole_arn   = data.terraform_remote_state.common.outputs.ecsInstanceRole_arn
+  ecsServiceRole_arn    = data.terraform_remote_state.common.outputs.ecsServiceRole_arn
+  cd_user_arn           = data.terraform_remote_state.common.outputs.codeship_arn
+  image_retention_count = 20
```

```suggestion
  image_retention_count = 20
  image_retention_tags = ["latest","develop"]
```
This ensures that images with these tags will always be retained, even if there are 20 newer images with other tags.